examples/federation-workshop/cluster-2/bundle/Federation.yaml

```diff
@@ -0,0 +1,23 @@
+# /*
+# |    Protect your secrets, protect your sensitive data.
+# :    Explore VMware Secrets Manager docs at https://vsecm.com/
+# </
+# <>/  keep your secrets… secret
+# >/
+# <>/' Copyright 2023–present VMware, Inc.
+# >/'  SPDX-License-Identifier: BSD-2-Clause
+# */
+
+apiVersion: spire.spiffe.io/v1alpha1
```

Comment:
ClusterFederatedTrustDomain needs to be manually set up during initial cluster setup.

This process can be scripted too, if needed.

Reply:
The trustdomainbundle won’t be empty normally. it’s just a placeholder.